test/verify/check-system-terminal

```diff
@@ -350,6 +350,68 @@ PROMPT_COMMAND='printf "\033]0;%s@%s:%s\007" "${USER}" "${HOSTNAME%%.*}" "${PWD/
         self.wait_line(2, "/var")
         self.assertIn("No such process", m.execute(f"kill -0 {pid} 2>&1 || true"))
 
+    def testExternalState(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go("/playground/terminal")
+
+        b.wait_text('.pf-v6-c-toggle-group__button.pf-m-selected', "Terminal 1")
+
+        # Run something in Terminal 1
+        b.wait_in_text(line_sel(1), "admin@local")
+        b.input_text("echo Terminal 1\n")
+        b.wait_text(line_sel(2), "Terminal 1")
+
+        # Switch to Terminal 2
+        b.click('.pf-v6-c-toggle-group__button:contains(Terminal 2)')
+        b.wait_text('.pf-v6-c-toggle-group__button.pf-m-selected', "Terminal 2")
+        b.wait_text_not(line_sel(2), "Terminal 1")
+
+        # Run something else there
+        b.wait_in_text(line_sel(1), "admin@local")
+        b.input_text("echo Terminal 2\n")
+        b.wait_text_not(line_sel(2), "Terminal 2")
+
+        # Switch back to Terminal 1, old output should still be there
+        b.click('.pf-v6-c-toggle-group__button:contains(Terminal 1)')
+        b.wait_text('.pf-v6-c-toggle-group__button.pf-m-selected', "Terminal 1")
+        b.wait_text(line_sel(2), "Terminal 1")
+
+        # Reset, this will output "disconnected" and start a new shell
+        # process
+
+        b.input_text("echo $$\n")
+        b.wait_in_text(line_sel(5), "admin@local")
+        old_pid = b.text(line_sel(4))
+
+        b.click('button:contains(Reset)')
+        b.click(".terminal")  # Focus the terminal again
+        b.wait_in_text(line_sel(5), "disconnected")
+        b.wait_in_text(line_sel(6), "admin@local")
```

Comment:
Interesting -- it doesn't clear the terminal on a disconnect/reset? Isn't that a bug?

Reply:
Yes, convinced, I'll reset the terminal in `resetChannel`.

Reply:
I don't have a strong opinion either way, but it feels slightly more correct to clear after a reset. Terminals can render all kinds of oddities, and they also have ANSI mode states and such, which may mess up the output.